### PR TITLE
Added bundle entry point which wraps all available entry points

### PR DESCRIPTION
**Pull request by midnightercz**
Also changed AppImageBuilder to produce output to the multi command bundle

### CODE REVIEW

**Review by querti — APPROVED** [review]

File: src/pubtools/sign/bundle.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from .signers.msgsigner import msg_clear_sign, msg_container_sign
+
+
+@click.group()
+def cli():
+    """Pubtools-sign multi command bundle."""
+    pass
+
+
+cli.add_command(msg_clear_sign, name="msg-clear-sign")
```
Comment (emilyzheng) [inline src/pubtools/sign/bundle.py:13]: msg_clear_sign_main and msg_container_sign_main are commands.

Comment (midnightercz) [reply src/pubtools/sign/bundle.py:13]: yeah, this needs to be rebased

Comment (midnightercz) [reply src/pubtools/sign/bundle.py:13]: or actually not, click wrappers are around methods without main

Comment (emilyzheng) [reply src/pubtools/sign/bundle.py:13]: Isn't click on *_main functions?
https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/msgsigner.py#L462
https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/msgsigner.py#L478

Functions without main are not commands.

**Pull request merged**